definitions/private-law/json/CaseEvent/Order/CaseEvent-nonprod.json

```diff
@@ -42,7 +42,7 @@
     "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/populate-draft-order-dropdown",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/submitted",
-    "PreConditionState(s)": "*",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS;SUBMITTED_PAID;CASE_ISSUED;JUDICIAL_REVIEW;ALL_FINAL_ORDERS_ISSUED;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
```

Comment:
Is this event required for AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS?

Reply:
@yogendraU  removed AWAITING_SUBMISSION_TO_HMCTS;AWAITING_RESUBMISSION_TO_HMCTS as per below confluence page
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1721438715